### DISCUSSION TITLE
Open with Kitty in new tab

### DISCUSSION THREAD

**Discussion by rhsev** [kovidgoyal/kitty#6830] [Q&A] [answered]
I can open text documents like Markdown in Kitty with "open with" (MacOS Sonoma). Is it possible that a new window is not opened every time, but that the document is opened in a new tab?
I have searched the configuration file and the documentation but found no information.
Thank you.

**Comment by kovidgoyal** [comment]
https://sw.kovidgoyal.net/kitty/remote-control/

**Reply by rhsev** [reply]
Thank you for your superfast answer. 
Doesn't this documentation refer to when I open kitty via the command line? 
I thought I would have to specify in the config file the parameters for kitty for how to start with "open with".

By the way: I think it's a great feature that this is possible at all to use "open with". I would like to open .md-files with Helix in Kitty.

**Comment by kovidgoyal** [comment, marked as answer]
you write a script that opens kitty however you like with remote
control. Then use that script as your open with application

**Reply by rhsev** [reply]
Thank you. Then that's what I do.
